Azure/azure-cli
# [Network] `az network bastion create`: Add new parameter `--scale-units` and `--sku` to support setting scale unit

> Closed https://github.com/Azure/azure-cli/issues/19375

**Description**

Add new parameter `--scale-units` and `--sku` to support setting scale unit.

**Testing Guide**

```
$ az network bastion create -g {rg} -n {bastion} --vnet-name {vnet} --public-ip-address {ip1} --scale-units {num} --tags a=b
```
---

This checklist is used to make sure that common guidelines for a pull request are followed.
